doc/source/contribution/documentation/introduction.md

```diff
@@ -144,6 +145,16 @@ Actually, the command to generate translation `.edit.po` files is the same as th
 Therefore, there's no need to memorize different commands for generating translation files and reflecting the translations.
 ```
 
+### Preview translations on HTML files
+
+You can preview your translations in your browser in HTML format. The step for reflecting translations into `.po` files also generates the corresponding HTML files. These files are located in `../groonga.doc/doc/${LANGUAGE}/html/`. Each file corresponds to a `.rst` or `.md` file. Open the generated HTML file in your web browser to review your translations.
```

Comment:
```suggestion
You can preview your translations in your browser in HTML format. The step for reflecting translations into `.po` files also generates the corresponding HTML files. These files are located in `../groonga.doc/doc/${LANGUAGE}/html/`. Each file corresponds to a `.rst` or `.md` file. Open the generated HTML file in your Web browser to review your translations.
```

In other places, it is written `Web browser`.

Reply:
fix: [4d6862f](https://github.com/groonga/groonga/pull/1751/commits/4d6862f83778ea4f410e1c070df74fd277650877) Thanks. I also added it because I missed adding the `Web` in other places too.